ExternalBeamPlanning/MRML/vtkMRMLRTObjectiveNode.cxx

```diff
@@ -0,0 +1,103 @@
+/*==============================================================================
+
+Copyright (c) Laboratory for Percutaneous Surgery (PerkLab)
+Queen's University, Kingston, ON, Canada. All Rights Reserved.
+
+See COPYRIGHT.txt
+or http://www.slicer.org/copyright/copyright.txt for details.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+This file was originally developed by Lina Bucher, Institut fuer Biomedizinische
+Technik at the Karlsruher Institut fuer Technologie (IBT-KIT) and German Cancer
+Research Center (DKFZ)
+
+==============================================================================*/
+
+// MRMLObjective includes
```

Comment:
There is no project called `MRMLObjective`. This is `ExternalBeamPlanning MRML`.